api/fsi_test.go

```diff
@@ -141,7 +141,7 @@ func TestNoHistory(t *testing.T) {
 	}
 
 	// Expected response for body of the dataset
-	expectBody = `{"data":{"path":"fsi_init_dir/body.csv","data":[["one","two",3],["four","five",6]]},"meta":{"code":200},"pagination":{"nextUrl":"/body/peer/test_ds?page=2"}}`
+	expectBody = `{"data":{"path":"fsi_init_dir/body.csv","data":[["one","two",3],["four","five",6]]},"meta":{"code":200},"pagination":{"page":1,"pageSize":50,"nextUrl":"/body/peer/test_ds?page=2","prevUrl":""}}`
```

Comment:
Shouldn't this also include result_count? Think -1 is good if it's "hard to get" or "N/A"

Reply:
Good point. I've filed #1374 as a follow up to set these. I'd like to do `pagination.resultsCount` in one big pass